metals/src/main/scala/scala/meta/internal/metals/JsonParser.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.internal.metals
+import com.google.gson.{Gson, JsonElement, JsonObject}
+
+import scala.reflect.{ClassTag, classTag}
+import scala.util.{Failure, Success, Try}
+
+sealed trait JsonParser {
+  implicit class Parsable(data: Any) {
```

Comment:
Adding a method on `Any` seems ify (it will be available on everything :fearful: ). Can we make it on a `JsonElement`? We can then add it to `MetalsEnrichments`. It will involve an additional step of matching, but I think it's cleaner.